public/javascripts/control.js

```diff
@@ -469,9 +471,22 @@
                         required: true,
                         value: {},
                         summary: false },
+        short:        { name: 'Short Label',
+                        type: 'uiText',
+                        description: 'If provided, a short label that is presented to the person filling the form on smaller screens.',
```

Comment:
Let's match https://getodk.github.io/xforms-spec/#languages: "A shorter version of the label, meant for very small screens, or to be shown in a summary of the form data."